controllers/memberstatus/memberstatus_controller_test.go

```diff
@@ -98,22 +98,22 @@ func TestOverallStatusCondition(t *testing.T) {
 
 	restore := test.SetEnvVarsAndRestore(t, test.Env(commonconfig.OperatorNameEnvVar, defaultMemberOperatorName))
 	defer restore()
-	nodeAndMetrics := newNodesAndNodeMetrics(
+	nodes, metrics := newNodesAndNodeMetrics(
```

Comment:
what is the reasoning behind splitting these two values?

Reply:
because nodes are obtained by the "default" client and metrics are obtained by the `AllNamespaces` client